<launch>

    <!--硬件启动-->
    <include file="$(find real_arm_hand)/launch/real_bring_up.launch"/>

    <!-- 抓取节点启动 -->
    <node name="plan_grasps" pkg="real_arm_hand" type="plan_grasps">
        <param name="move_group_arm" value="arm" />
        <param name="move_group_gripper" value="gripper" />
    </node>

    <!--yolo启动-->
    <node name="ros_yolo" pkg="rs_yolo" type="rstest.py">
       <!-- <param name="move_group_arm" value="arm" />-->
    </node>

</launch>